webrtc.html

```diff
@@ -6245,7 +6252,7 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
     readonly        attribute RTCIceRole           role;
     readonly        attribute RTCIceComponent      component;
     readonly        attribute RTCIceTransportState state;
-    readonly        attribute RTCIceGatheringState gatheringState;
+    readonly        attribute RTCIceGathererState  gatheringState;
```

Comment:
Could RTCIceGathererState be RTCIceTransportGatheringState to keep the word "gathering", but be explicit that it belongs to the RTCIceTransport?